package.json

```diff
@@ -15,31 +15,45 @@
   },
   "bugs": "https://github.com/ForestAdmin/toolbelt/issues",
   "dependencies": {
-    "@forestadmin/context": "1.3.0",
+    "@forestadmin/context": "1.12.0",
     "@oclif/command": "1.5.4",
     "@oclif/config": "1.8.8",
     "@oclif/errors": "1.2.2",
     "@oclif/plugin-help": "2.1.3",
     "@oclif/plugin-not-found": "1.2.4",
     "@oclif/plugin-warn-if-update-available": "1.7.0",
+    "app-root-path": "3.0.0",
+    "atob": "^2.1.2",
```

Comment:
Hardcoded versions are expected 👍 
A few other dependencies below have the same `^`.